src/api/ai_help.rs

```diff
@@ -269,6 +272,9 @@ fn log_errors_and_record_response(
     user_id: i64,
     help_ids: HelpIds,
 ) -> Result<Option<mpsc::UnboundedSender<CreateChatCompletionStreamResponse>>, ApiError> {
+    if !history_enabled {
+        return Ok(None);
+    }
```

Comment:
We check for `history_enabled` below (line 300), so I don't understand why we return early here?

Reply:
Otherwise we spawn a thread and gather the full answer just to do nothing. Yes, we should also refactor the code below. But that was the important piece.

Reply:
Extracted to https://github.com/mdn/rumba/pull/438.